.coveragerc

```diff
@@ -1,4 +1,26 @@
 [run]
+branch = True
+source = .
```

Comment:
Better use `_pytest,testing` here - less to omit then.

Reply:
hmm but then you have to have a special snowflake `.coveragerc` for each project

Reply:
?

I think it is OK for a project to have its own sources defined here - after all you would then have special omits instead.

Reply:
🤷‍♂️ no special `omit`s here -- I copied this from one of my projects

Reply:
Still different than "defaults", no? :)